API/authentication/.env.sample

```diff
@@ -1,2 +1,7 @@
-    COGNITO_USERPOOLID='us-east-2_POOL'
-    COGNITO_CLIENTID='CLIENTID'
\ No newline at end of file
+COGNITO_USERPOOLID='us-east-2_POOL'
+COGNITO_CLIENTID='CLIENTID'
+DB_HOST=''
```

Comment:
Are we leaving these blank?

Reply:
This is a blank template so that one can make their own .env, the actual .env is not in source control (.gitignore) for security reasons.